src/main/java/game/map/ShowMap.java

```diff
@@ -0,0 +1,35 @@
+package game.map;
+import game.pojo.Continent;
+import game.pojo.Country;
+import java.util.List;
+public class ShowMap {
+    public static void showMap(Map map) {
+        System.out.println("---------------------------Continents---------------------------");
+        List<Continent> continents = map.getContinents();
+        for (Continent continent : continents) {
+            System.out.println(continent.getName());
+        }
+        System.out.println("---------------------------Countries---------------------------");
+        List<Country> countries = map.getCountries();
+        for (Country country : countries) {
+            System.out.println(country.getName());
+            System.out.println("Borders:");
```

Comment:
Why to use the term borders?
(as they used neighbors in the description - https://users.encs.concordia.ca/~paquet/wiki/images/6/6f/Build1Grading.SOEN6441.2024.4.pdf)

Reply:
You can use the below to print all the neighboring countries to a country.

```suggestion
            for (Country country : countries) {
                System.out.println(country.getName());
                System.out.println("Neighbors:");
                for (Country neighbor : country.getNeighbors) {
                    System.out.println("- " + neighbor.getName());
                }
            }
```